src/main/java/com/softserveinc/ita/pageobjects/admin/FacultiesPage.java

```diff
@@ -0,0 +1,28 @@
+package com.softserveinc.ita.pageobjects.admin;
+
+import com.codeborne.selenide.ElementsCollection;
+import com.codeborne.selenide.SelenideElement;
+
+import java.util.List;
+
+import static com.codeborne.selenide.Selenide.$$x;
+import static com.codeborne.selenide.Selenide.$x;
+
+public class FacultiesPage extends MainMenu {
+    SelenideElement searchField = $x("//div/input[@id = 'mat-input-2']");
```

Comment:
If the variable is used only once, class variable creation is not required, use directly in the method